### ISSUE TITLE
Custom Template

### ISSUE THREAD

**Issue by jankout** [terminal42/contao-ajaxform#18]
Hello, 

I would appreciate to be able to add custom template(s). Could you integrate this possibility of the core Contao function? That would be great. Thank you.

**Comment by aschempp** [comment]
I don't think this is currently possible easily, since the extension uses multiple templates (for ajax and confirmation) and does not follow the Contao naming conventions (since it's both a module and content element). Very similar to the form module I guess.
